nixos/doc/manual/release-notes/rl-2003.xml

```diff
@@ -148,13 +148,6 @@
       Now, all bash code is held to the same high standard, and the rather complex stateful manipulation of the options can be discarded.
     </para>
    </listitem>
-   <listitem>
```

Comment:
Mybe mention instead that `slim` is now your `slim-ng`

Reply:
Release notes should not be undone